hypervisor/arch/x86/guest/virq.c

```diff
@@ -432,7 +432,13 @@ int32_t acrn_handle_pending_request(struct acrn_vcpu *vcpu)
 			}
 		}
 
-		acrn_inject_pending_intr(vcpu, pending_req_bits, injected);
+		/*
+		 * DO NOT inject interrupts when emulating the split-lock.
```

Comment:
Defer injection of interrupt to be after MTF VM exit, when emulating the split-lock.